src/utils/InputValidators.js

```diff
@@ -189,7 +190,7 @@ export function createExcludesValidator(values, errorMsg) {
     let exists;
     if (typeof(values) === 'object' && values instanceof Set) {
       exists = values.has(value);
-    } else if (Array.isArray(values)) {
+    } else if (Array.isArray(values) || List.isList(values)) {
```

Comment:
what is we simplify this whole thing more?
```es6
return (values?.has(value) || values?.includes(value)) && (errorMsg || translate('duplicate.error', value)); 
```

Reply:
I like that idea, and it does the right thing with sets and arrays, but it does not work correctly with the immutable stuff. All immutable collections, including Maps, Lists, and Sets have both a 'has' and an 'include', and using the right one depends on which of these classes are being used. I'll adjust this to handle those special cases and then default to your suggestion.